src/services/recim/activity.ts

```diff
@@ -66,9 +66,10 @@ const createActivity = (newActivity: UploadActivity): Promise<CreatedActivity> =
 
 const getActivityByID = (ID: number): Promise<Activity> => http.get(`recim/activities/${ID}`);
 
-const getAllActivities = (active: boolean, time: String): Promise<Activity[]> => {
+const getActivities = (active: boolean, time: String): Promise<Activity[]> => {
   if (time) return http.get(`recim/activities?active=${active}&time=${time}`);
-  return http.get(`recim/activities?active=${active}`);
+  else if (active) return http.get(`recim/activities?active=${active}`);
```

Comment:
```suggestion
  if (active) return http.get(`recim/activities?active=${active}`);
```
else is redundant on return